sqlglot/dialects/trino.py

```diff
@@ -3,6 +3,7 @@
 from sqlglot import exp
 from sqlglot.dialects.dialect import merge_without_target_sql, trim_sql
 from sqlglot.dialects.presto import Presto
+from sqlglot.tokens import TokenType
```

Comment:
We should make these changes in presto.py, because `TDIGEST` is also [supported](https://prestodb.io/docs/current/language/types.html#t-digest) by Presto.